src/minimizer_mapper.cpp

```diff
@@ -2161,7 +1926,30 @@ vector<pair<bool, bool>> mapping_was_rescued;
              assert(minimizer.hits<=hard_hit_cap) ;
          }
     }
-    cerr << "\t" << uncapped_mapq << "\t" << fragment_cluster_cap << "\t" << mapq_score_groups[0] << "\t" << mapq_explored_caps[0] << "\t" << new_cluster_cap << "\t" << mappings.first.front().mapping_quality();  
+    cerr << "\t" << uncapped_mapq << "\t" << fragment_cluster_cap << "\t" << mapq_score_groups[0] << "\t" 
+         << mapq_explored_caps[0] << "\t" << new_cluster_cap << "\t" << mappings.first.front().mapping_quality() << "\t";  
+    for (size_t i = 0 ; i < scores.size() ; i++) {
+        pair<pair<size_t, size_t>, pair<size_t, size_t>> indices = pair_indices[i];
+        Alignment& aln_1 = alignments[indices.first.first].first[indices.first.second];
+        Alignment& aln_2 = alignments[indices.second.first].second[indices.second.second];
+
+        int64_t dist = distances[i];
+        assert(dist == distance_between(aln_1, aln_2)); 
```

Comment:
Isn't this going to slow us down to recompute the distance just to check?

I guess it's protected by the print_minimizers `ifdef` though.